### DISCUSSION TITLE
How to hide the new thing at the top ?

### DISCUSSION THREAD

**Discussion by KaKi87** [coder/code-server#6536] [Q&A] [answered]
Hi,

I'm sorry if this question has been asked many times already but I really can't find it after browsing recent issues and discussions, and the fact that I can't find the thing's name doesn't help.

This :

![](https://github.com/coder/code-server/assets/21284089/3c4d8855-b7e3-4f45-b40a-9d0d1de41333)

Thanks

**Comment by code-asher** [comment]
I think they are calling this the "title bar", or at least that is the name of the whole bar that contains that search input.  Unfortunately I do not see a way to disable it nor the input, not even with zen mode.

Supposedly if you use it as a PWA it will merge with the browser title bar so it takes up less space but I am not sure.

**Reply by KaKi87** [reply]
Well, then I guess I'll just downgrade until it can be disabled, because it's really a significant vertical space loss.
In which version did it appear ?
Thanks

**Reply by code-asher** [reply]
I think Microsoft added it in 1.82.0 of VS Code, which was included in 4.17.0 of code-server, so you will want to downgrade to 4.16.1 or lower.

**Reply by code-asher** [reply, marked as answer]
Oh wait I think I found a way.  I was reading the changelog: https://code.visualstudio.com/updates/v1_82#_command-center-now-on-by-default

Apparently you can right click and uncheck "command center" which is the name of that search input (there is also a setting for it).  When I tried this the entire title bar went away.
